service.cli/execute.sh

```diff
@@ -13,21 +13,88 @@ echo "..."
 echo
 # ----------------------------------------------------------------
 # This script shall be modified according to the needs in order to run the service
-# The inputs defined in ${INPUT_FOLDER}/inputs.json are available as env variables by their key in capital letters
-# For example: input_1 -> $INPUT_1
 
-# put the code to execute the service here
-# For example:
-env
-ls -al "${INPUT_FOLDER}"
+# From isolve: checking for resources limit env vars injected by osparc
+SIMCORE_NANO_CPUS_LIMIT="${SIMCORE_NANO_CPUS_LIMIT:-0}"
+echo "Setting/getting CPU limit gives this: ${SIMCORE_NANO_CPUS_LIMIT}"
+if [ "${SIMCORE_NANO_CPUS_LIMIT}" -ne "0" ]
+then
+    echo "Found NANO_CPU limits: ${SIMCORE_NANO_CPUS_LIMIT}"
```

Comment:
@mguidon, mainly for curiosity:
Before I had:
`echo "$INFO" "Found NANO_CPU limits: ${SIMCORE_NANO_CPUS_LIMIT}"`
But I had an `unbound variable` error in my local oSPARC deploy.

I had to remove "$INFO" to make it work. I am not sure if this is due to something that has changed with the log levels.